app/stac_api/views.py

```diff
@@ -364,11 +364,19 @@ def get_queryset(self):
         )
 
     def get(self, request, *args, **kwargs):
-        if (not Collection.objects.filter(name=self.kwargs['collection_name']).exists() or
-                not Item.objects.filter(name=self.kwargs['item_name']).exists()):
-            logger.error("Either the collection (%s) or the item (%s) do not exist",
-                         self.kwargs['collection_name'], self.kwargs['item_name'])
+        if not Collection.objects.filter(name=self.kwargs['collection_name']).exists():
+            logger.error("The collection, %s, does not exist", self.kwargs['collection_name'])
             raise Http404("This collection does not exists.")
```

Comment:
Sorry to be very picky but as these error messages go to customer, here I would also add the collection name as above. This improves the user experience, if the user does a typo, he can directly see it in the error message. So I suggest to use something like: `The collection {self.kwargs['collection_name']} does not exists.`

Reply:
No problem. Next correction incoming.